scripts/us_epa/air_emissions_inventory/national_state/process.py

```diff
@@ -228,21 +231,24 @@ def _parse_source_files(self):
             None
         """
         source_file_to_method_mapping = {
-            "national_tier1_caps": self._national_emissions,
-            "state_tier1_caps": self._state_emissions
+            "state_tier1_caps_05Apr2023_Ktons": self._state_emissions,
+            "national_tier1_caps_05Apr2023": self._national_emissions
```

Comment:
Does this apply to data from other years? I wonder if the dictionary should still include the lines from the previous version of this file.

Reply:
yes in this same source file previous and current year (2022) data is available in the below files.
1. https://www.epa.gov/system/files/other-files/2023-04/national_tier1_caps_05Apr2023.xlsx
2. https://www.epa.gov/system/files/other-files/2023-04/state_tier1_caps_05Apr2023_Ktons.xlsx